example.js

```diff
@@ -1,16 +0,0 @@
-var hstring = require('./index')
```

Comment:
We have this exact example in the README, and it was two examples to maintain.

Also, it was failing standard with a lot of "unhandled error" linting problems and I didn't want to fix them.

If you wish, I can add back this example and add an `eslint-disable handle-callback-error` comment at the top (just like the tests).